markdownlint.js

```diff
@@ -321,11 +328,22 @@ function lintAndPrint(stdin, files) {
   printResult(lintResult);
 }
 
-if ((files.length > 0) && !options.stdin) {
-  lintAndPrint(null, diff);
-} else if ((files.length === 0) && options.stdin && !options.fix) {
-  const getStdin = require('get-stdin');
-  getStdin().then(lintAndPrint);
-} else {
-  program.help();
+function main() {
+  if ((files.length > 0) && !options.stdin) {
+    lintAndPrint(null, diff);
+  } else if ((files.length === 0) && options.stdin && !options.fix) {
+    const getStdin = require('get-stdin');
+    getStdin().then(lintAndPrint);
+  } else {
+    program.help();
+  }
+}
+
+if (require.main === module) {
```

Comment:
Why add this?

Reply:
Ah, this is my Python background leaking out. I guess it doesn't really matter if you never expect anyone to import this as a module. Will lift this out.